src/main/java/org/jabref/logic/ai/AiDefaultPreferences.java

```diff
@@ -10,6 +10,7 @@
 public class AiDefaultPreferences {
     public enum PredefinedChatModel {
         GPT_4O_MINI(AiProvider.OPEN_AI, "gpt-4o-mini", 128000),
+        GPT_LOCAL(AiProvider.GPT4ALL, "Phi-3.1-mini-128k-instruct-Q4_0-precise-output-tensor.gguf", 128000),
```

Comment:
It's better to leave it empty like in Hugging Face models. You can rename `PredefinedChatModel.HUGGING_FACE` to blank and use it.

Oh, and when you will do this, also add a prompt text with your model into "chat model" text field (like in Hugging Face variant).

Did you understood what I mean? I might wrote it too quickly

Reply:
![image](https://github.com/user-attachments/assets/845c2ea1-8f62-44f4-9fe8-f5d3eb73ee07)
prompt text has been added.

Reply:
Great!